### DISCUSSION TITLE
修改 vi 之后导致很多其他的 vi 需要保存

### DISCUSSION THREAD

**Discussion by LemonSugar198** [ruanqizhen/labview_book#78] [Q&A]
因为在使用 git 管理代码，有时候只是打开工程修改了一个子 vi，却要保存很多其他没有改动的 vi，这时候就会造成很多冲突，我知道应该是这个被改动的子 vi 在很多代码中使用了，但是如果避免这种编程呢？

**Comment by ruanqizhen** [comment]
可以考虑一下可执行代码与源代码分开保存：
<https://lv.qizhen.xyz/optimization_mechanism#可执行代码与源代码分离>

**Reply by LemonSugar198** [reply]
原来要这样，学到了，谢谢阮老师
